generators/app/index.js

```diff
@@ -198,7 +198,10 @@ module.exports = yeoman.Base.extend({
         {filter: function (name) {
           return (name.indexOf('login') == -1);
         }});
-
+    
+     fse.copySync(this.jhipsterHome+ '/src/main/webapp/app/admin/admin.state.js','./app/main/jhipster/admin/admin.state.js');
+     fse.copySync(this.jhipsterHome+ '/src/main/webapp/app/admin/tracker','./app/main/jhipster/admin/tracker');
```

Comment:
Add ` if (this.enableWebsocket) {` around files specific to projects with websockets